base/Base/Base_Base_implementation.F90

```diff
@@ -206,9 +209,12 @@ module subroutine MAPL_FieldAllocCommit(field, dims, location, typekind, &
              allocate(VR8_1D(UNGRID(1)), STAT=STATUS)
              _VERIFY(STATUS)
              VR8_1D = INIT_VALUE
-             call ESMF_FieldEmptyComplete(FIELD, farrayPtr=VR8_1D,    &
+             call ESMF_FieldEmptyComplete(FIELD, farray=VR8_1D,    &
+                  indexflag=ESMF_INDEX_DELOCAL, &
                   datacopyFlag = ESMF_DATACOPY_REFERENCE,             &
                   gridToFieldMap=gridToFieldMap,                      &
+                  ungriddedLBound=[1],&
+                  ungriddedUBound=[ungrid(1)], &
                   rc = status)
```

Comment:
Again - please check status.

Reply:
Satisfied in 224 at the end of the 'if' block

Reply:
Yes, but ... this is somewhat confusing to the reader.  They then have to go back and see how many different places that failure could have come from.  Best to do it right away (in source) unless there is a pressing reason to not do so.